extensions/amp-image-slider/0.1/test/integration/test-amp-image-slider.js

```diff
@@ -446,5 +494,97 @@ config.run('amp-image-slider', function() {
             return pollByBarLeftPos(rect.right);
           });
     });
+
+    it('should seekTo correct position', () => {
+      return prep()
+          .then(() => {
+            const button = doc.querySelector('#button');
+            const pos10Percent = rect.left + 0.1 * rect.width;
+            button.click();
+            return pollByBarLeftPos(pos10Percent);
+          });
+    });
+
+    it('should show hint again on scroll back and into viewport', () => {
+      let slider;
+      let container_;
+      let hint;
+      let mouseDownEvent;
+      return prep()
+          .then(() => {
+            slider = doc.querySelector('#slider');
+            container_ = slider
+                .querySelector('.i-amphtml-image-slider-container');
+            hint = slider.querySelectorAll('.i-amphtml-image-slider-hint')[0];
+            mouseDownEvent = createFakeEvent(
+                'mousedown',
+                0,
+                0
+            );
+            container_.dispatchEvent(mouseDownEvent);
+            return poll('should receive hidden class', () => {
```

Comment:
Can we use [`MutationObserver`](https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver) here?

Reply:
Per offline discussion, we'll do this in a separate PR.